index.js

```diff
@@ -66,7 +66,55 @@ function toggleColor(element) {
         element.style.color = "green"
     }
 }
+const changeColor = document.querySelector("#header")
+changeColor.addEventListener('click', function (event){
```

Comment:
Good! But a few things:
- We have the header variable already defined earlier in the file.
- All we need to do is add an event listener to that header. We provide a callback function to be executed when our "click" event gets triggered.
- The callback function we can pass in to the event listener is already made for us!. We just need to pass in the event target as an argument to the function.
```
header.addEventListener("click", function(e){
    toggleColor(e.target)
})
```